src/image/image.py

```diff
@@ -42,12 +42,10 @@ def __init__(
         uint8_tensor, float32_tensor = get_tensor_from_np_array(self.np_array)
         self.uint8_tensor = uint8_tensor.to(self.device)
         self.float32_tensor = float32_tensor.to(self.device)
+        self.width = uint8_tensor.shape[2]
```

Comment:
Is there a way to make a unit test for this? Or at least a way to manually test that the fix works?